internal/pkg/runtime/engine/apptainer/process_linux.go

```diff
@@ -45,6 +45,7 @@ import (
 	apptainercallback "github.com/apptainer/apptainer/pkg/plugin/callback/runtime/engine/apptainer"
```

Comment:
please close lock file descriptor in `StartProcess` function to not expose it to container and keep a useless reference that would hold the lock